### PR TITLE
docs: Update HIP-32 to note removal of ability to set account's alias with an update

### PR DESCRIPTION
**Pull request by david-bakin-sl**
**Description**:

Update HIP-32 to note removal of ability to set account's alias with an update

Refers to the section of HIP-583 that does this.

### CODE REVIEW

File: HIP/hip-32.md [hunk 1]
```diff
@@ -57,7 +57,9 @@ An automatically-created account will be created with only one public key (the o
 
 At most one account can ever have a given alias. If a transaction auto-creates the account, any further transfers to that alias will simply be deposited in that account, without creating anything, and with no creation fee being charged.
 
-An account created normally has no alias. It can be given an alias with an account update, but only if that alias is not currently used by any other account, and only if the update transaction is signed by the private key corresponding to the public key in the alias.
+An account created normally has no alias. ~~It can be given an alias with an account update, but only if that alias is not currently used by any other account, and only if the update transaction is signed by the private key corresponding to the public key in the alias.~~
+* **UPDATE:** This ability has been removed by [HIP-583](https://hips.hedera.com/hip/hip-583#update-account-with-alias),
```
Comment (Nana-EC) [inline HIP/hip-32.md:61]: Should we be clear that it was never implemented but the goal was rather given up on after exploring HIP 583 and 631?
```suggestion
* **UPDATE:** This intent has been revisited and retracted by [HIP-583](https://hips.hedera.com/hip/hip-583#update-account-with-alias),
```

**Review by Nana-EC — COMMENTED** [review]
LG, 1 comment

File: HIP/hip-32.md (hunk 1 above)
Comment (david-bakin-sl) [reply HIP/hip-32.md]: TY I like your wording.

**Review by Nana-EC — APPROVED** [review]

**Review by lukelee-sl — APPROVED** [review]
LGTM

**Comment by david-bakin-sl** [comment]
@mgarbs - ready for your review

**Pull request merged**